huaweicloud/services/iam/data_source_huaweicloud_identity_group.go

```diff
@@ -0,0 +1,176 @@
+package iam
+
+import (
+	"github.com/chnsz/golangsdk/openstack/identity/v3/groups"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
+)
+
+func DataSourceIdentityGroup() *schema.Resource {
+	return &schema.Resource{
+		Read: DataSourceIdentityGroupV3Read,
+		Schema: map[string]*schema.Schema{
+
```

Comment:
A blank line can make our code more readable, bug it is not recommended to keep this blank line.

Reply:
fixed